contrib/completion/bash/netctl

```diff
@@ -0,0 +1,463 @@
+#! /bin/bash
+
+GLOBAL_COMMANDS="\
```

Comment:
Must we specify these options explicitly, can we get it from reading things themselves? Same comment for GLOBAL_OPTIONS. I am worried about maintaining this file along with any change as we go forward. I hopt we don't add too many global vars and options, but having to specify at two places doesn't seem good.